### PR TITLE
Update Get-SmbMultichannelConnection.md

### PR DESCRIPTION
**Pull request by msbemba**
updated description for SMB instance and examples

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/winserver2012-ps/smbshare/Get-SmbMultichannelConnection.md [hunk 1]
```diff
@@ -127,7 +126,7 @@ Contoso-SO     False          172.30.182.10  192.168.101.11 32             12
 Contoso-SO     False          172.30.182.10  192.168.101.11 32             12             False          False
 ```
 
-This example retrieves the all of the connections made between the SMB client network interfaces and the SMB server network interfaces for the SMB server named Contoso-SO.
+This command retrieves the all of the connections made between the SMB client network interfaces and the SMB server network interfaces for the SMB server named Contoso-SO.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/smbshare/Get-SmbMultichannelConnection.md]:
```suggestion
This command retrieves all of the connections made between the SMB client network interfaces and the SMB server network interfaces for the SMB server named Contoso-SO.
```

Comment (e0i) [reply docset/winserver2012-ps/smbshare/Get-SmbMultichannelConnection.md]: cc @msbemba

**Pull request merged**